keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1162,6 +1163,18 @@ Once all of those steps are complete, kubelet will notice that the claims are
 ready and run the pod. Until then it will keep checking periodically, just as
 it does for other reasons that prevent a pod from running.
 
+### Handling non graceful node shutdowns
+
+When a node is shut down unexpectedly and is tainted with an `out-of-service`
+taint with NoExecute effect as explained in the [Non graceful node shutdown KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown),
+all running pods on the node will be deleted by the GC controller and the
+resources used by the pods will be deallocated. However, they will not be
+un-prepared as the node is down and Kubelet is not running on it.
+
+Resource drivers should be able to handle this situation correctly and
+should not expect `UnprepareNodeResources` to be always called before
+`Deallocate`.
```

Comment:
I think this sentence saying that `UnprepareNodeResources` may be omitted, but can be read as "they can be called in opposite order". Please make it more explicit what sentence means

Reply:
Thank you for pointing out to this. Re-frazed, PTAL.